dotCMS/src/main/webapp/dotcms-webcomponents/p-vjjfne0x.system.entry.js

```diff
@@ -0,0 +1 @@
),r.status=t.status,r);throw e;case 2:return[2,t.json()]}}))}))})).then((function(t){var r=t.entity;e.runSuccessCallback(r)})).catch((function(t){var r=t.message,s=t.status;e.errorMessage=getErrorMessage(r)||fallbackErrorMessages[s]}))};class_1.prototype.runSuccessCallback=function(contentlet){var successCallback=this.getSuccessCallback();if(successCallback){return function(){return eval(successCallback)}.call({contentlet:contentlet})}};class_1.prototype.getSuccessCallback=function(){var t=getFieldsFromLayout(this.layout).filter((function(t){return t.variable==="formSuccessCallback"}))[0];return t.values};class_1.prototype.resetForm=function(){var t=this;var e=Array.from(this.el.querySelectorAll("form dot-form-column > *"));e.forEach((function(e){return __awaiter(t,void 0,void 0,(function(){var t;return __generator(this,(function(r){switch(r.label){case 0:r.trys.push([0,2,,3]);return[4,e.reset()];case 1:r.sent();return[3,3];case 2:t=r.sent();console.warn(""+e.tagName,t);return[3,3];case 3:return[2]}}))}))}))};class_1.prototype.getUpdateValue=function(){return getFieldsFromLayout(this.layout).filter((function(t){return t.fixed===false})).reduce((function(t,e){var r;var s=e.variable,a=e.defaultValue,n=e.dataType,o=e.values;return Object.assign(Object.assign({},t),(r={},r[s]=a||(n!=="TEXT"?o:null),r))}),{})};class_1.prototype.getMaxSize=function(t){var e=__spreadArrays(t.target.attributes);var r=e.filter((function(t){return t.name==="max-file-length"}))[0];return r&&r.value};class_1.prototype.uploadFile=function(t){var e=this;var r=new DotUploadService;var s=t.detail.value;var a=this.getMaxSize(t);var n=t.target;if(!a||s.size<=a){this.uploadFileInProgress=true;n.errorMessage="";return r.uploadFile(s,a).then((function(t){e.errorMessage="";n.previewImageUrl=t.thumbnailUrl;n.previewImageName=t.fileName;e.uploadFileInProgress=false;return t})).catch((function(t){var r=t.message,s=t.status;n.clearValue();e.uploadFileInProgress=false;e.errorMessage=getErrorMessage(r)||fallbackErrorMessages[s];return null}))}else{n.reset();n.errorMessage="File size larger than allowed "+a+" bytes";return Promise.resolve(null)}};Object.defineProperty(class_1.prototype,"el",{get:function(){return getElement(this)},enumerable:true,configurable:true});Object.defineProperty(class_1,"watchers",{get:function(){return{layout:["layoutWatch"],fieldsToShow:["fieldsToShowWatch"]}},enumerable:true,configurable:true});Object.defineProperty(class_1,"style",{get:function(){return"dot-form{display:block}dot-form>form label{margin:0;padding:0}dot-form>form dot-form-column>*{display:block;margin:2rem 0}dot-form>form dot-form-column>:first-child{margin-top:0}dot-form>form dot-form-column>:last-child{margin-bottom:0}dot-form>form .dot-form__buttons{display:-ms-flexbox;display:flex;-ms-flex-direction:row;flex-direction:row;-ms-flex-pack:end;justify-content:flex-end}dot-form>form .dot-form__buttons button:last-child{margin-left:1rem}"},enumerable:true,configurable:true});return class_1}())}}}));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid assignments in operands](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5062311)